### DISCUSSION TITLE
layui-layout框体布局中layui-footer页脚会遮挡layui-body主体内容

### DISCUSSION THREAD

**Discussion by 237198352** [layui/layui#1628] [问答] [answered]
如题，官网中的框体布局例子中，当我删掉内容主体区域最下方两个`<br><br>`标签时，底部固定区域layui-footer会遮挡layui-body主体内容
![图片](https://github.com/layui/layui/assets/115967131/955ecd7c-19d2-4565-b98f-c7ae6b9d688e)
当更换主体内容后，可能需要加多个`<br>`标签才能将主体内容全部显示：
![图片](https://github.com/layui/layui/assets/115967131/e15066fe-8a49-40c2-8efe-bbc32c69d1b5)
当换成2个`<br>`标签时效果如下：
![图片](https://github.com/layui/layui/assets/115967131/ba390ec6-c4d1-49e6-9ba8-a14c35702cac)
倒是不影响使用，但感觉怪怪的。大家有遇到这个问题么？

**Comment by sentsim** [comment]
由于底部栏是 `fixed` 定位，因此主体内容底部区域被遮挡难以避免。建议在底部元素中设置 `padding-bottom` 规避。

**Comment by 237198352** [comment, marked as answer]
在`layui-body`里添加样式：`margin-bottom:45px;`解决了该问题
